src/Drupal/MediaTrait.php

```
@@ -73,6 +73,9 @@ public function mediaRemoveType(string $type): void {
    * @Given the following media :media_type exist:
    */
   public function mediaCreate(string $media_type, TableNode $nodesTable): void {
+    // Delete entities before creating them.
+    $this->mediaDelete($media_type, $nodesTable);
```

PHPMD (2.15.0)

The variable $media_type is not named in camelCase. (undefined)

(CamelCaseVariableName)